app/views/layouts/base.html.slim

```diff
@@ -13,6 +13,8 @@ html lang="en-US"
     
     / TODO: Find a better location for this (only used in loan criteria charts)
     = javascript_include_tag "https://www.gstatic.com/charts/loader.js"
+    javascript:
+      google.charts.load('current', {'packages':['corechart']});
```

Comment:
Inside the page where you can edit the loan questionnaire, all issues related to displaying the chart mentioned in the above comment are addressed,